peachjam/templates/peachjam/saved_document/_update.html

```diff
@@ -16,7 +16,7 @@
 <div hx-swap-oob="true"
      id="saveDocumentModalDialog-{{ saved_document.document.id }}"
      class="modal-dialog">
-  <div class="modal-content">
+  <div class="modal-content text-start">
```

Comment:
Is this extra class necessary?

Reply:
Yes. The modal on the saved documents page aligns right without it. It inherits the styles of the edit and delete icons.